docs/how-to-guides/running-feast-in-production.md

```diff
@@ -144,3 +185,142 @@ prediction = model.predict(feature_vector)
 It is important to note that both the training pipeline and model serving service only needs read access to the feature registry and associated infrastructure. This prevents clients from accidentally making changes to the feature store.
 {% endhint %}
 
+## 4. Retrieving online features for prediction
+
+Once you successfully loaded (or in Feast terminology materialized) your data from Batch Source into Online Store you can start consuming features for inference phase.
+There are three approaches for that purpose sorted from the most simple one (in operational sense) to the most performant:
+
+### 4.1. Python SDK reads directly from Online Store
+
+This approach is the most convenient if you want to keep your infrastructure as minimalistic as possible and avoid deploying extra services.
+Python SDK will connect directly to the Online Store (Redis, Datastore, etc), pull the raw data and run transformations locally (if required).
+The obvious drawback is that your service (running the prediction) must be written on Python to use Feast Python SDK. 
+But since a lot of Data Science tools are already using Python, and there's a great chance your service will be using it as well, this approach is still pretty valid for production.
```

Comment:
```suggestion
But since a lot of data science tools are already using Python, and there's a great chance your service will be using it as well, this approach is still pretty valid for production.
```